site/index.md

```diff
@@ -26,6 +26,7 @@ or use the download icon in the upper-right corner of each tutorial.
 maxdepth: 1
 ---
 
+content/tutorial-style-guide
```

Comment:
The style guide seems developer/contributor oriented. Why does it need to be included in the rendered site?

Reply:
Agreed - I think it would be best to separate out the contributor/developer info into it's own category separate from the pedagogical material itself; xref #85